torch/onnx/symbolic_opset9.py

```diff
@@ -393,6 +393,9 @@ def squeeze(g, self, dim=None):
 
 
 def prelu(g, self, weight):
+    self_sizes = self.type().sizes()
+    if self_sizes and len(self_sizes) > 2:
+        weight = g.op("Unsqueeze", weight, axes_i=list(range(1, len(self_sizes) - 1)))
```

Comment:
also we already supported the broadcasting in ONNX's PRelu, why do we want to unsqueeze it again?

Reply:
The reason is https://github.com/pytorch/pytorch/issues/21271. The shape of weight should be [channels, 1, 1] according to [ONNX document](https://github.com/onnx/onnx/blob/master/docs/Operators.md#PRelu) (ONNX PReLU requires unidirectional broadcasting), however, the actual shape is [channels] instead.